R/db.R

```diff
@@ -297,6 +297,14 @@ write_data_safely <- function(dbconn, tblname, df, key, safe_columns) {
   }
 
   write_column_hashed_data <- function(df, tblname, append = TRUE) {
+    ## Don't cache anything that is in the blacklist.
+    if (!is.null(blacklist)) {
+      if (any(is.na(blacklist))) {
+        df <- df[apply(df, 1, function(x) all(!(is.na(x)))),]
+      }
+      df <- df[apply(df, 1, function(x) all(!(x %in% blacklist))),]
+    }
+
```

Comment:
Oh, I see. Which means we won't cache the entire row that has this blacklisted value. Fair enough.